biophi/humanization/web/views.py

```diff
@@ -206,14 +206,21 @@ def humanize_results_get(task_id):
 def humanize_detail_get(task_id, index):
     result_task_id = scheduler.get_result_task_id(task_id, index)
     result: HumanizeAntibodyTaskResult = scheduler.get_result(task_id, index)
+
+    len_results = scheduler.get_results_len(task_id)
+    
+    has_prev_page = int(index) > 1
+    has_next_page = int(index) < len_results
```

Comment:
@hrifanov can you please do this comparison inside the template? then we wouldn't need to pass these variables

for example `{% if result_index > 1 %}` instead of `{% if has_prev_page %}`